pkgs/games/osu-lazer/default.nix

```diff
@@ -1,111 +1,68 @@
-{ lib, stdenv, fetchFromGitHub, fetchurl, makeWrapper, makeDesktopItem, linkFarmFromDrvs
-, dotnetCorePackages, dotnetPackages, cacert
-, ffmpeg_4, alsa-lib, SDL2, lttng-ust, numactl, alsa-plugins
+{ lib
+, stdenvNoCC
```

Comment:
Why `stdenvNoCC` instead of `stdenv`? I don't see the necessity here.

Reply:
Just to avoid pulling in the compiler, i figured we might as well.